### ISSUE TITLE
Bugs and suggestions for the POI-Editor dialog

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/Osmand#2806]
**Suggestion-1:**
In the **Advanced** tab, OsmAnd lists _all_ tags (and _only_ tags).
So this tag should be named "Tags".

**BUG-1:**
In the **Advanced** tab, OsmAnd lists each tag and its value in white boxes.
If I add a new attribute, it shows tag+values pair (e.g. "shoes:women") in **Tag** field, and the **Value** field is left blank. This leaves the user under wrong impression that the value of the tag is not set yet.
**Desired:** The "tag:value" pair should be split and populated in their respective fields.

**BUG-2:**
If I tap on the **Cancel** button, OsmAnd pops up a dialog:

**Are you sure?**
Any unsaved changes will be lost. Continue? 
Cancel     OK

The user may be confused when he sees two "cancel"s consecutively. 

- What does "Cancel" actually mean: Cancel the previous cancel, or cancel the POI-editing?
- What does "continue" mean: Continue editing, or continue on your way out?

**Solution:** Instead of "Cancel/OK" buttons, show "Return to POI / Quit".

**Comment by matthijskooijman** [comment]
> In the Advanced tab, OsmAnd lists all tags (and only tags). So this tab should be named "Tags".

I disagree here. Calling this "advanced" has the extra benefit of marking this tab as more complicated than the rest of the interface, meaning you should only use it when you know what you're doing. If you name it "tags", users might relate that to previous experience with tagging and think they can just add tags freely without any reservation.

> BUG-1:

With the current version of OsmAnd, it seems that new tags added and existing tags to be edited in the same style, so I think this one is no longer relevant.

> BUG-2:
> (confusing cancel prompt)

Agreed, a Yes/No prompt would be better, or "Discard / Back" or something more explicit.
